applications/add_unit_to_parameter_in_db.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a unit to parameters in the DB.
+
+    This application should not be used by anyone but expert users and only in extreme cases.
+    Therefore, no additional documentation about this applications will be given.
+
+"""
+
+import logging
+
+import simtools.config as cfg
+import simtools.util.commandline_parser as argparser
+import simtools.util.general as gen
+from simtools import db_handler
+
+
+def main():
+
+    parser = argparser.CommandLineParser(description=("Add a unit field to a parameter in the DB."))
+    parser.initialize_default_arguments(add_workflow_config=False)
+
+    args = parser.parse_args()
+    cfg.setConfigFileName(args.configFile)
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+
+    # parsToUpdate = ["altitude"]
+    parsToUpdate = ["ref_long", "ref_lat"]
+
+    # units = ["m"]
+    units = ["deg", "deg"]
```

Comment:
Oh, now I understand. The changes are hardwired? So you hardly ever expect to this be used?